freud/diffraction.py

```diff
@@ -24,15 +23,6 @@
 import freud.util
 from freud.util import _Compute
 
-_HAS_MPL = find_spec("matplotlib") is not None
-if _HAS_MPL:
-    import matplotlib.cm
-    import matplotlib.colors
-
-    import freud.plot
-else:
-    msg_mpl = "Plotting requires matplotlib."
-
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.